packages/firebase_ml_vision/example/test_driver/text_recognizer.dart

```diff
@@ -22,4 +25,36 @@ void textRecognizerTests() {
       expect(recognizer.close(), completes);
     });
   });
+
+  group('Cloud $TextRecognizer', () {
+    final TextRecognizer recognizer =
```

Comment:
nit: type is optional

Reply:
Could you please elaborate?

Reply:
`final recognizer = FirebaseVision.instance.cloudTextRecognizer();`